scripts/import-styles/main.ts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env -S deno run -A
+import usercssMeta from "usercss-meta";
+import { ensureDir } from "std/fs/mod.ts";
+import { walk } from "std/fs/walk.ts";
+import { parse as parseFlags } from "std/flags/mod.ts";
+import { join } from "std/path/mod.ts";
+
+import { REPO_ROOT } from "@/deps.ts";
+
+const flags = parseFlags(Deno.args, { boolean: ["fix"] });
+const subDir = flags._[0]?.toString() ?? "";
+const stylesheets = walk(join(REPO_ROOT, "styles", subDir), {
```

Comment:
We don't need to parse flags here.
```suggestion
import usercssMeta from "usercss-meta";
import { ensureDir } from "std/fs/mod.ts";
import { walk } from "std/fs/walk.ts";
import { join } from "std/path/mod.ts";

import { REPO_ROOT } from "@/deps.ts";

const stylesheets = walk(join(REPO_ROOT, "styles"), {
```